syne_tune/optimizer/schedulers/transfer_learning/zero_shot.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import logging
+from typing import Dict, Optional
+
+import numpy as np
+import pandas as pd
+import xgboost
+
+from benchmarking.blackbox_repository.blackbox_surrogate import BlackboxSurrogate
+from syne_tune.config_space import Domain
+from syne_tune.optimizer.schedulers.searchers.searcher import BaseSearcher
+from syne_tune.optimizer.schedulers.transfer_learning import TransferLearningTaskEvaluations, TransferLearningMixin
+
+logger = logging.getLogger(__name__)
+
+
+class ZeroShotTransfer(TransferLearningMixin, BaseSearcher):
+    def __init__(
+            self,
+            config_space: Dict,
+            transfer_learning_evaluations: Dict[str, TransferLearningTaskEvaluations],
+            metric: str,
+            mode: str = 'min',
+            sort_transfer_learning_evaluations: bool = True,
+            use_surrogates: bool = False,
+            random_seed: Optional[int] = None
+    ) -> None:
+        """
+        A zero-shot transfer hyperparameter optimization method which selects configurations that minimize the average
```

Comment:
I would propose the following change:
```suggestion
        A zero-shot transfer hyperparameter optimization method which jointly selects configurations that minimize the average
```
since it may feel otherwise that you are just greedily picking the next top one.